web/assets/show-hide-case-summary.js

```diff
@@ -0,0 +1,28 @@
+export default function showHideCaseSummary(prefix) {
+  const button = document.querySelector(
+    '[data-module="app-case-summary-toggle"]',
+  );
+  const buttonText = document.querySelector(".case-summary-toggle-text");
+  const buttonIcon = document.querySelector(".case-summary-toggle-icon");
+  const caseSummary = document.querySelector(".case-summary");
+
+  if (button && buttonText && buttonIcon && caseSummary) {
+    button.onclick = () => {
```

Comment:
Better to use `addEventListener` to allow each button to have multiple events:
```suggestion
    button.addEventListener('click', () => {
```